cloudsql/cloudsqlpod.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Generally we don't recommend users creating bare pods directly. Can you wrap this in a deployment or RC with replicas=1?